test/Transport/SendmailTest.php

```diff
@@ -223,4 +223,38 @@ public function testTrimmedParameters()
 
         $this->assertSame('-R hdrs', $r->getValue($this->transport));
     }
+
+    public function testAllowMessageWithEmptyToHeaderButHasCcHeader()
+    {
+        $message = new Message();
+        $message->addCc('matthew@zend.com')
+                ->setSender('ralph.schindler@zend.com', 'Ralph Schindler')
+                ->setSubject('Testing Zend\Mail\Transport\Sendmail')
+                ->setBody('This is only a test.');
+
+        $this->transport->send($message);
+    }
+
+    public function testAllowMessageWithEmptyToHeaderButHasBccHeader()
+    {
+        $message = new Message();
+        $message->addBcc('zf-crteam@lists.zend.com', 'CR-Team, ZF Project')
+                ->setSender('ralph.schindler@zend.com', 'Ralph Schindler')
+                ->setSubject('Testing Zend\Mail\Transport\Sendmail')
+                ->setBody('This is only a test.');
+
+        $this->transport->send($message);
+    }
+
+    public function testDonotAllowMessageDoesnotHasToAndCcAndBccHeader()
```

Comment:
Please fix test method name, I think it should be:
`testDoNotAllowMessageWithoutToAndCcAndBccHeaders`

Reply:
done